supported-services.md

```
@@ -0,0 +1,87 @@
+
+While efforts have been made to ensure accuracy, no responsibility is taken for errors or omissions.
+
+| Name | Free? | Comment |
+| --- | --- | --- |
+| ✅ [Argos](https://translate.argosopentech.com/) | yes | Implemeted to be used locally with libretranslate (API of Argos) |
+| ✅ [Google Translate](https://cloud.google.com/translate/docs/reference/rest) | no  | Uses Cloud Translation API |
+| ✅ [Lingvanex](https://lingvanex.com/) | yes |     |
+| ✅ [modernMT](https://www.modernmt.com/) | yes |     |
+| ✅ [Microsoft Translator](https://www.microsoft.com/en-us/translator/) | no  | Quite low request limits |
+| ✅  [PONS](https://de.pons.com/p/ubersetzungsapi) | no  | No latin and serbian in contrast to the [PONS online dictionary](https://de.pons.com/%C3%BCbersetzung) |
+| ✅ [Reverso](# "reverso.com") | yes | Sometimes no API response |
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)